snippets/React (JSX).cson

```diff
@@ -31,6 +31,10 @@
     prefix: "_cdm"
     body: "componentDidMount() {\n\t${1}\n}${2}"
 
+  "React: componentDidCatch(error, info) { ... }":
+    prefix: "_cdc"
+    body: "componentDidUpdate(error, info) {\n\t${1}\n}${2}"
```

Comment:
@mclark4386 looks like the body won't be what you're expecting

Reply:
oh man... derp, sorry